oddt/fingerprints.py

```diff
@@ -0,0 +1,173 @@
+
+"""
```

Comment:
it should be a one-line docstring (see [pep257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings))